src/test/java/dk/dbc/updateservice/actions/UpdateRequestActionTest.java

```diff
@@ -253,7 +253,7 @@ void testValidRecordForUpdate() throws Exception {
         assertThat(updateRequestAction.performAction(), is(ServiceResult.newOkResult()));
```

Comment:
Generelt om denne testklasse: AssertActionUtil bruges mange andre steder til at checke at child actions er de korrekte, f.eks. i klassen oven for (UpdateOperationActionTest). Det ville være rart hvis UpdateRequestActionTest gjorde det på samme måde.

Reply:
Hæ, det tog mig lige et par minutter at hitte ud af at du havde glemt et s mellem n og U :-|
Jo, det kunne nok gøre det bedre og der er flere andre ting jeg også har lyst til at gøre i update testcases - nogle steder endda indsætte kommentarer så man ved hvad der foregår. Måske vi skulle lave en story på f.eks. 3 points og så tæve løs på dem - tror det vil være bedre end vores drypvise tilpasning.